docs/features/multidomain.rst

```diff
@@ -88,18 +88,25 @@ domain of a router, if and only if one of the above conditions matches.
 Switching the domain
 --------------------
 
-**via commandline**:
+Via commandline
+^^^^^^^^^^^^^^^
 
 ::
 
-    uci set gluon.core.domain="newdomaincode"
-    gluon-reconfigure
-    reboot
+    gluon-switch-domain 'newdomaincode'
 
-**via config mode:**
+When the node is not in config mode, ``gluon-switch-domain`` will automatically
+reboot the node by default. This can be suppressed by passing ``--no-reboot``::
 
-To allow switching the domain via config mode, ``config-mode-domain-select``
-has to be added to GLUON_FEATURES in the site.mk.
+    gluon-switch-domain --no-reboot 'newdomaincode'
+
+Switching the domain without reboot is currently **experimental**.
```

Comment:
I think this statement is key as it makes very clear that we currently recommend rebooting to safely change the domain. This also expresses the need for more testing in the wild, which is apparently somewhat involved or @RalfJung would've already done so.

Reply:
I am completely fine with requiring a reboot. We saw the previous problem even with the reboot method, just because I waited a minute during the 3-command sequence that changes the domain.  But then gluon extensions (hoodselector, domain-director) should also do these reboots, and I am not sure if they do.

Reply:
The domain director should definitely be moved to the new commands as soon as this PR is merged.

The scheduled-domain-switch is changed as part of this PR and reboots for the change. The hoodselector uses `--no-reboot`, but as the whole hoodselector is experimental, using an experimental command seems fine to me.

Reply:
> The domain director should definitely be moved to the new commands as soon as this PR is merged.

I can take care of that.

Reply:
Proposed the change here https://git.darmstadt.ccc.de/ffda/firmware/packages/-/merge_requests/4